src/meltano/core/tracking/iglu-client-embedded/schemas/com.meltano/cli_context/jsonschema/1-1-0

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Snowplow context describing a Meltano cli command",
+    "self": {
+        "vendor": "com.meltano",
+        "name": "cli_context",
+        "format": "jsonschema",
+        "version": "1-1-0"
+    },
+    "type": "object",
+    "properties": {
+        "command": {
+            "description": "",
+            "type": "string",
+            "maxLength": 1024
+        },
+        "parent_command_hint": {
```

Comment:
we can infer it from list of the contexts, but thought it was handy to just explicitly include a ref to what the upstream parent was.